docs/source/index.rst

```diff
@@ -144,6 +144,13 @@ Welcome to Kedro's documentation!
 
    logging/logging
 
+.. toctree::
+   :maxdepth: 2
+   :caption: Kedro in your workflow
```

Comment:
"Kedro in your workflow" sounds a bit weird, what about "Integrations"? (And rename the directory to `integrations`).

Reply:
Which makes me think that maybe we could add more pages here, like Kedro + Pandera, Kedro + MLFlow 👀 (essentially describing how to use the appropriate plugins)

Reply:
> Which makes me think that maybe we could add more pages here, like Kedro + Pandera, Kedro + MLFlow 👀 (essentially describing how to use the appropriate plugins)

Yes! That's the plan 🙂

> "Kedro in your workflow" sounds a bit weird, what about "Integrations"? (And rename the directory to integrations).

What do you think @stichbury? I like this change.

Reply:
Awesome, made this change.